src/IdBroker.php

```diff
@@ -29,7 +29,7 @@ class IdBroker extends Component implements PersonnelInterface
     /**
      * @var IPBlock[]
      */
-    public $trustedIpRanges = [];
+    public $validIpRanges = [];
```

Comment:
As with the other PR with a similar change, you may also want to update the README.md accordingly.

Reply:
updated